src/test/datascience/interactiveWindow.vscode.common.test.ts

```diff
@@ -238,52 +223,6 @@ for i in range(10):
         );
     });
 
-    test('LiveLossPlot', async () => {
-        const code = `from time import sleep
```

Comment:
We already have tests for this (this is basically display updates & multi-mime types),
I'm taking this out as I dont think IW should focus on testing core functionality in the Kernel.
Just slows tests

Reply:
@amunger  @rebornix /cc